Chw.html

```diff
@@ -505,7 +505,7 @@ <h2 class="text-center">CDM CHW Form v 0.01</h2>
 <div class="section">
     <div class="form-group">
         <label for="encounter.provider_id_select">Provider Name:</label>
-        <input class="form-control" id="encounter.provider_id_select" type="text"
+        <input class="form-control validProviderOnly" id="encounter.provider_id_select" type="text"
```

Comment:
I'm guilty of not doing this early in the development of the form, but I think moving forward we should do this :)